landing-pages/site/content/en/blog/airflow-survey-2020/index.md

```diff
@@ -0,0 +1,426 @@
+our users are data engineers, scientists or analysts. The 2020 results are similar to [those from 2019](https://airflow.apache.org/blog/airflow-survey/) with
+visible slight increase in ML use cases.
+
+Additionally, 79% of users uses Airflow on daily basis and 16% interacts with it at least once a week.
+
+**How many people work in your company?**
+
+|        |   No. |     % |
+|--------|-------|-------|
+| 200+   |   107 | 52.71 |
+| 51-200 |    44 | 21.67 |
+| 11-50  |    37 | 18.23 |
+| 1-10   |    15 |  7.39 |
+
+**How many people in your company use Airflow?**
+
+|       |   No. |     % |
+|-------|-------|-------|
+| 1-5   |    84 | 41.38 |
+| 6-20  |    75 | 36.95 |
+| 21-50 |    23 | 11.33 |
+| 50+   |    21 | 10.34 |
+
+Airflow is a software that is used and trusted by big companies. We can also see that Airflow can work
+fine for teams of different sizes. However, in some cases users may use multiple Airflow instances.
+
+
+**Are you considering moving to other workflow engines?**
+
+|                               |   No. |     % |
+|-------------------------------|-------|-------|
+| No, we are happy with Airflow |   174 | 85.71 |
+| Yes                           |    29 | 14.29 |
+
+Nearly 1 out of 7 users is considering migrating to other workflow engines. Their decision is usually
+justified by need of **easier workflow writing experience** (12.32%), **better UI/UX** and **faster scheduler**
+(8.37% both).
+
+While the first point may be addressed by (TaskFlow API)[http://airflow.apache.org/docs/apache-airflow/stable/concepts.html#taskflow-api] in Airflow 2.0 the other two are definitely addressed
+in the new major version. And the early feedback from 2.0 users seems to be confirming it.
+
+The alternative engines considered by users are mainly Prefect and Argo. Some participants also mentioned
+Luigi, Kubeflow or custom solutions.
+
+
+**Are you or your team actively participating in Airflow development - contributing?**
+
+|                 |   No. |     % |
+|-----------------|-------|-------|
+| I wish we could |    99 | 48.77 |
+| No              |    59 | 29.06 |
+| Yes             |    45 | 22.17 |
+
+This is really heart-warming result. It means that 1 out of 5 users contributes actively to our project!
+But it would be good to learn if there's something else than time that is stopping people who wish to contribute
+from doing it. If there are some other obstacles we definitely would like to learn about them so we can improve.
+That said - if you know something we can improve please reach out via Slack, dev list or Github
+discussions.
+
+**How likely are you to recommend Apache Airflow?**
+
+|               |   No. | 2020 % | 2019 % |
+|---------------|-------|--------|--------|
+| Very Likely   |   125 | 61.58  | 45.45% |
+| Likely        |    62 | 30.54  | 40.26% |
+| Neutral       |    11 |  5.42  | 10.71% |
+| Unlikely      |     3 |  1.48  | 2.60%  |
+| Very unlikely |     2 |  0.99  | 0.97%  |
+
```

Comment:
Do we also have the counts for 2019? It would be helpful to see the count in comparison to 45% as well as the 125/61.58% comparisson.

Reply:
Imho the count will be misleading as there was more responses last year, so using % gives us a common base line